packages/terra-site/src/examples/form/examples/Select.jsx

```diff
@@ -7,9 +7,9 @@ import Select from 'terra-form/lib/Select';
 const SelectExamples = () => (
   <form>
     <Select
-      choices={['Puppies', 'Kittens', 'Snappers', 'Bumblers', 'Joeys', 'Microprocessors']}
+      choices={['wow', 'this', 'that']}
```

Comment:
options? This could be what is causing the forms page to fails to load in the heroku instance.